blocks/banner/banner.css

```diff
@@ -1,19 +1,19 @@
-body > main > div.section.banner-container {
+.banner-container {
   background-color: var(--primary);
   color: var(--secondary);
   padding: 52px 100px;
 }
 
-body > main > div.section.banner-container > div > div > div > div > h4 {
+h4 {
```

Comment:
now h4 is global because the css is not scoped. either
- move h4 styling to styles.css
- or scope h4 (nest it inside .banner-container)